src/Constants/commands_messages.py

```diff
@@ -14,6 +12,8 @@
 CREATE_PROFILE_NAME_MESSAGE = "What would you like to name your profile?: "
 CREATE_PROFILE_SUCCESS_MESSAGE = "Successfully created profile. Now switching to profile: {}"
 EXISTING_NAME_MESSAGE = "A {} with the name '{}' already exists."
+FOOD_OPEN_ERROR_MESSAGE = "Nothing is currently open. You must open a 'fridge' or 'freezer' " + \
```

Comment:
I think a better name would be like `FOOD_STORAGE_NOT_OPEN_ERROR_MESSAGE` or something

Reply:
fixed!